ee/packages/omnichannel-services/src/OmnichannelTranscript.ts

```diff
@@ -232,8 +232,20 @@ export class OmnichannelTranscript extends ServiceClass implements IOmnichannelT
 					continue;
 				}
 
-				const fileBuffer = await uploadService.getFileBuffer({ file: uploadedFile });
-				files.push({ name: file.name, buffer: fileBuffer, extension: uploadedFile.extension });
+				try {
+					const fileBuffer = await uploadService.getFileBuffer({ file: uploadedFile });
+					files.push({ name: file.name, buffer: fileBuffer, extension: uploadedFile.extension });
+				} catch (e: unknown) {
+					this.log.error(`Failed to get file ${file._id}`, e);
+					// Push empty buffer so parser processes this as "unsupported file"
+					files.push({ name: file.name, buffer: null });
+
+					if ((e as Error).message === 'MAX_PAYLOAD_EXCEEDED') {
```

Comment:
Ideally, we should not tie this to the NATS error message, but since we don't have an abstraction for this, we could put a TODO comment to refactor this when it's possible. WDYT?